### ISSUE TITLE
Show the best font colour to use with a picked background colour

### ISSUE THREAD

**Issue by stsdc** [cassidyjames/palette#47]
Would be cool to display also a font colour to use with picked background colour for the best contrast and style.

**Comment by cassidyjames** [comment]
This is hard because it could suggest pure white or black (which it basically does in the UI without explicitly saying it), but more often you might want to use it with a lighter or darker shade of the same color. I don't think we have official guidelines for foreground colors to use with elementary palette background colors, but that could be interesting to do upstream.

**Comment by stsdc** [comment]
I created this issue because I saw, that on [official HIG](https://elementary.io/pl/docs/human-interface-guidelines#color) You are not using just black colour for foreground. I thought it was standardized.
![obraz](https://user-images.githubusercontent.com/6031763/77943323-52195a00-72bd-11ea-8ce8-0c1d0fd1f1b2.png)
Here was used `#330000` instead of black.

**Comment by cassidyjames** [comment]
@stsdc yeah that isn't standardized or documented anywhere, but we could.

**Comment by cassidyjames** [comment]
@stsdc I've opened an issue at https://github.com/elementary/website/issues/2331

**Comment by hanaral** [comment]
elementary/website#2330 was fıxed, I assume it would be possible now
